### DISCUSSION TITLE
Endpoint to get database alerts / notifications?

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#18257] [General]
Hi,

is there an endpoint I can use to get a list of all non dismissed alerts / notifications, I can only see the WebSockets watch endpoint for notifications, but ideally need some way to just poll periodically for any new notifications.

Or is it possible to do this through the changes API?

Thanks

**Comment by arekpalinski** [comment]
Hi Nick,

Currently we just expose the _count_ of alerts (and performance hints) through SNMP and `/databases` endpoint.

Exposing the actual content of those via an endpoint is an option. Could you tell us more about your use case?

**Reply by nickchampion** [reply]
Hi @arekpalinski I just want to be able to monitor the server for ETL related errors that occur occasionally but are not easily visible without logging onto the server dashboard

**Reply by arekpalinski** [reply]
I have opened https://issues.hibernatingrhinos.com/issue/RavenDB-22176/Expose-alerts-notifications-via-GET-endpoint

**Reply by arekpalinski** [reply]
Fixed in https://github.com/ravendb/ravendb/pull/18318. @nickchampion please have a look
